e2e/nomostest/clean.go

```diff
@@ -271,16 +271,24 @@ func isTestLabel(key string) bool {
 func deleteConfigSyncAndTestAnnotationsAndLabels(nt *NT, ns *corev1.Namespace) error {
 	var annotations = make(map[string]string)
 	var labels = make(map[string]string)
+	found := false
 	for k, v := range ns.Annotations {
 		if !isConfigSyncAnnotation(k) && !isTestAnnotation(k) {
 			annotations[k] = v
+		} else {
+			found = true
 		}
 	}
 	for k, v := range ns.Labels {
 		if !isConfigSyncLabel(k, v) && !isTestLabel(k) {
 			labels[k] = v
+		} else {
+			found = true
 		}
 	}
+	if !found {
```

Comment:
Why not check the length of `annotations` and `labels`?

Reply:
that might be more clear to read, yeah.

Reply:
updated